versioned/tiered/mongodb/pom.xml

```diff
@@ -35,12 +35,6 @@
       <artifactId>nessie-versioned-tiered-impl</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
```

Comment:
Do you know why this was done? Given that this is a Rocks patch, we should probably only change Rocks related items unless it's clearly denoted.

Reply:
I'm really not sure where this crept in. I have reverted this change.